### PR TITLE
Better test suite, fix for non-V8 browsers

### PR DESCRIPTION
**Pull request by justmoon**
This adds a bunch more tests (from 98 to 473) and fixes an issue where errors could not be constructed in browsers that don't use V8.

Depends on interledger/js-oer-utils#1

### CODE REVIEW

File: src/types/prefix-sha256.js [hunk 1]
```diff
@@ -41,33 +41,20 @@ class PrefixSha256 extends BaseSha256 {
    * Each prefix condition builds on an existing condition which is provided via
    * this method.
    *
-   * @param {Condition} subcondition Condition that will receive the prefixed
-   *   message.
+   * @param {Condition|String} subcondition Condition object or URI string
+   *   representing the condition that will receive the prefixed message.
```
Comment (emschwartz) [inline src/types/prefix-sha256.js]: Can you add a test for this?

**Comment by emschwartz** [comment]
Overall LGTM but I don't really know enough about it to 👍 commit https://github.com/interledger/five-bells-condition/pull/53/commits/748238a1d69e904bf493454debe2766696ffad50

File: src/types/prefix-sha256.js (hunk 1 above)
Comment (justmoon) [inline src/types/prefix-sha256.js]: Done

**Comment by justmoon** [comment]
Going to merge. Regarding the RSA code, it should be reviewed in general. @msierchio has offered to do it.

**Pull request merged**